activecampaign.php

```diff
@@ -59,7 +59,7 @@ function activecampaign_form_source($settings, $form, $static = false) {
 		if ($static) {
 			$source .= "static=1&";
 		}
-		$source .= sprintf("id=%d", $form["id"]);
+		$source .= sprintf("id=%d&%s", $form["id"], strtoupper(uniqid()));
```

Comment:
Should this be a version number or fingerprint instead of uniqid? This would allow the browser to continue using the cache until a new version number or fingerprint was applied.